mail_tracking/static/src/xml/mail_tracking.xml

```diff
@@ -54,7 +54,7 @@
             <t t-foreach="message.partner_trackings" t-as="tracking">
                 <t t-set="isCc" t-value="false" />
                 <t t-foreach="message.email_cc" t-as="cc">
-                    <t t-if="cc[0] == tracking[4]" t-set="isCc" t-value="true" />
+                    <t t-if="!tracking[1] and cc[0] == tracking[4]" t-set="isCc" t-value="true" />
```

Comment:
```suggestion
                    <t t-if="!tracking[1] && cc[0] === tracking[4]" t-set="isCc" t-value="true" />
```

This is JS!

Reply:
Sure! works because qweb replace it: https://github.com/odoo/odoo/blob/12.0/addons/web/static/lib/qweb/qweb2.js#L33

Reply:
Wow! I was wondering why it worked... Maybe it's to avoid things such as `&amp;&amp;`... Then better leave it as it is.